src/components/description/Description.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import expect from 'expect';
+import Description from './Description';
+
+
+configure({ adapter: new Adapter() });
+
+function assertElementHasText(wrapper, query, text) {
+  expect(wrapper.find(query).text()).toEqual(text);
```

Comment:
What does this do if there are multiple matches to the query?

Reply:
It looks like `.find()` will return every node matching the query https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/find.html